### ISSUE TITLE
on_touch_down() in FloatLayout() works in Kivy v.2.0.0 but not in v.2.1.0

### ISSUE THREAD

**Issue by HeRo002** [kivy/kivy#7861]
**Software Versions**
* Python: v3.8.9
* OS: Android v8.1.0 (on a MOTO G5 phone)
* Kivy: v2.1.0
* Kivy installation method: ??

**Describe the bug**
I just know that my customized version of the Kivy TouchTracer demo works with Kivy v. 2.0.0 but not with v. 2.1.0

**Expected behavior**
The problem is that the moment you touch the screen, the whole screen turns blue - like the famous MS-Windows "blue screen of death"...  :-) At least on my Android ver. 8.1.0 MOTO G5 phone.

**To Reproduce**
You can find a more or less minimal example that reproduces the problem here: https://groups.google.com/g/kivy-users/c/cDwRkQkZTA4

There you can also find the output of:
$ buildozer android run logcat >../touch-log.txt

As far as I can tell, the first 950 lines aren't interesting.

Robert added: 
"In that last log I can see where Android fails, about 7 seconds after Python/Kivy has initialized. But I have no idea why."

**Code and Logs and screenshots**
See https://groups.google.com/g/kivy-users/c/cDwRkQkZTA4

**Additional context**
I used the following commands in an Ubuntu Terminal while having the MOTO G5 phone connected via USB:
$ cd /<the directory of the above Python/Kviy example>
$ buildozer init
((I edited the buildozer.spec file - included above.))
$ buildozer android debug deploy run
$ adb logcat -c
$ buildozer android run logcat >../touch-log.txt

**Comment by misl6** [comment]
Hi @HeRo002 ,

I was doing some cleanup, and this issue doesn't seem to persist on `Kivy==2.2.0rc1`.
Feel free to re-open the issue if we missed something.

**Comment by HeRo002** [comment]
I just tried compiling my GeoESP Training app again with:

 I python  : [INFO   ] [Kivy        ] v2.2.1
 I python  : [INFO   ] [Python      ] v3.10.10 (main, Sep 19 2023, 21:40:45) [Clang 14.0.6 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0

But my Touchtracer still doesn't work. :-( 
So the question is which Kivy and Python versions I should try?
The purpose is that I have upgraded from Android API 30 to 34.
